.travis.yml

```diff
@@ -3,3 +3,9 @@ before_install:
   - sudo apt-get install -y python-jsonschema python-pil
 
 script: python ./validate.py
+
+after_success: if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then ./i18n/push_translations.sh; fi
```

Comment:
Even if it's currently quite clear what the both conditions check, but I think we should add a comment here